### ISSUE TITLE
Error sweeping wallet `requirement failed: Not all OutPoints belong to this wallet, diff`

### ISSUE THREAD

**Issue by Christewart** [bitcoin-s/bitcoin-s#4258]
Seen on 6b3be779 . When trying to sweep the Alice laptop from the bitcoin 2022 wallet.

```
2022-04-13T13:20:18UTC INFO [DLCWallet$DLCWalletImpl] Sending to bc1qee5thya9x8al98mkfmc3qrmlut96dlvqkdh3ga at feerate 1 sats/vbyte
2022-04-13T13:20:18UTC INFO [Server] Unhandled error in server:
java.lang.IllegalArgumentException: requirement failed: Not all OutPoints belong to this wallet, diff Vector(TransactionOutPoint(a96e3211c67a764d7fdaaaa82877300af4150199d2f7ebc33305a82062e839b8:0))
	at scala.Predef$.require(Predef.scala:337)
	at org.bitcoins.wallet.Wallet.$anonfun$sendFromOutPoints$3(Wallet.scala:468)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:467)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:63)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:100)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:94)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:100)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:49)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:48)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
```

**Comment by Christewart** [comment]
This got fixed after rescanning. Not sure what the root cause was with this is.

**Comment by Christewart** [comment]
fixed in #4290 / #4296
